src/test/unit/controls/PairedResult/PairedResultPanning-spec.js

```diff
@@ -98,7 +98,7 @@ describe("PairedResult", () => {
                     datelineGroup.getAttribute("transform")
                 ).translate;
                 expect(toNumber(translate[0], 10)).toBeGreaterThanOrEqual(80);
-                expect(toNumber(translate[1], 10)).toBeCloseTo(5);
+                expect(toNumber(translate[1], 10)).toBeCloseTo(2);
```

Comment:
Is this same for all the graph types? Maybe we can create a constant in commonHelpers and then reference that?

Reply:
Added here https://github.com/cerner/carbon-graphs/pull/107/commits/f960f072648a74aeb7a781ed0bc5cf91020a16e6

Reply:
I meant in the `commonHelpers` [here](https://github.com/cerner/carbon-graphs/blob/master/src/test/unit/helpers/commonHelpers.js)

Reply:
Done here https://github.com/cerner/carbon-graphs/pull/107/commits/f1b827cc6c61574d07c286ad6744c5da18bb9e90.